zerver/lib/events.py

```diff
@@ -1299,6 +1303,19 @@ def _draft_update_action(i: int) -> None:
             user_status.pop(user_id_str, None)
 
         state["user_status"] = user_status
+    elif event["type"] == "user_topic":
+        if event["visibility_policy"] == 0:
```

Comment:
Replace `0` with the `UserTopic.MUTED` type value that this corresponds to; maybe we need a user `UserTopic.NO_VISIBILITY_POLICY` constant equalling 0 here if we're going with this convention (I have other questions about it).

Reply:
Here, `0` corresponds to a `default` state as suggested by @gnprice [here]( https://chat.zulip.org/#narrow/stream/378-api-design/topic/stream.20IDs.20in.20.60muted_topics.60/near/1325661 ).  We send it while unmuting a topic. `UserTopic.MUTED` is represented by `1`.
As of now, we have only two possible states, the `default` and the `muted` state. We can add a `UserTopic.DEFAULT` choice.

Reply:
@timabbott I am unable to understand `VISIBILITY_POLICY_INHERIT` clearly. Here, `0` is supposed to represent the `default` state, i.e., no special user-topic relationship. Using a similar constant instead of hardcoding `0` sounds right though.

Reply:
Yeah, exactly, this is just a code readability thing. A visibility policy of 0 would never be stored in the database, but giving it a readable name for what `0` means in this context would be very helpful for anyone seeing the code to not feel like they need to look up what it means.